internal/testing/utils/path_test.go

```diff
@@ -0,0 +1,38 @@
+package path
+
+import (
+	"fmt"
+	"io/fs"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+// TestRootPathFromCWD exercises RootPathFromCWD and RootPath
+func TestRootPathFromCWD(t *testing.T) {
+	r, err := RootPathFromCWD()
+	require.NoError(t, err)
+	_, err = os.Stat(filepath.Join(r, "LICENSE"))
+	require.NoError(t, err, "Must find a LICENSE file")
+
+	// Ensure there are no other license files in sub-directories
+	err = filepath.WalkDir(r, func(p string, d fs.DirEntry, err error) error {
+		if err != nil {
+			return err
+		}
+		// Skip the root of the projuct
```

Comment:
projuct -> projuctitron

Reply:
Thanks.
Interesting that codespell and linter didn't catch that, right?

Fixed c92e04a